### PR TITLE
Initial project: Alien invasion

### PR DESCRIPTION
**Pull request by logiflo**
Add a initial pygame application. It also implements the ship and its movement.

### CODE REVIEW

File: AlienInvasion/alien_invasion.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    """Initialize game and create a screen object
+    """
+    pygame.init()
+    ai_settings = Settings()
+    screen = pygame.display.set_mode(
+        (ai_settings.screen_width, ai_settings.screen_height))
+    pygame.display.set_caption("Alien Invasion")
+
+    # Make the ship
+    ship = Ship(ai_settings, screen)
```
Comment (feserr) [inline AlienInvasion/alien_invasion.py:18]: Why sending the screen for set the position of the ship? It should be better to use a X and Y variables.

File: AlienInvasion/game_functions.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        elif event.type == pygame.KEYDOWN:
+            check_KEYDOWN(event, ship)
+
+        elif event.type == pygame.KEYUP:
+            check_KEYUP(event, ship)
+
+    return True
+
+
+def update_screen(ai_settings, screen, ship):
```
Comment (feserr) [inline AlienInvasion/game_functions.py]: This should be named as "draw_screen".

File: AlienInvasion/ship.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        self.screen = screen
+        self.ai_settings = ai_settings
+
+        # Load the ship image and get its rect.
+        self.image = pygame.image.load('images/ship3.png')
+        self.rect = self.image.get_rect()
+        self.screen_rect = self.screen.get_rect()
+
+        # Start each new ship at the bottom center of the screen.
+        self.rect.centerx = self.screen_rect.centerx
```
Comment (feserr) [inline AlienInvasion/ship.py:20]: Use the X and Y variables as told before.

**Review by feserr — CHANGES_REQUESTED** [review]

File: AlienInvasion/alien_invasion.py (hunk 1 above)
Comment (logiflo) [reply AlienInvasion/alien_invasion.py:18]: I pass the screen as an argument to keep the ship on the screen.

**Review by feserr — APPROVED** [review]

**Pull request merged**